depends/packages/packages.mk

```diff
@@ -1,5 +1,5 @@
-zcash_packages := libsodium rustcxx utfcpp
-packages := boost libevent zeromq $(zcash_packages) googletest tl_expected
+zcash_packages := libsodium rustcxx utfcpp tl_expected
+packages := boost libevent zeromq $(zcash_packages) googletest
```

Comment:
Does this mean you now know the difference between these two lists? Because I would like to …

Oh, I have a guess – zcash_packages are ones that are not managed in Bitcoin?